electrode-ota-server-dao-plugin/src/dao.js

```diff
@@ -345,7 +343,7 @@ export default class DaoExpressCassandra {
       for (let i = 0; i < packages.length; i++) {
         let pkg = packages[i];
         let tagsDoMatch = matchTags(tags, pkg.tags);
-        let versionsDoMatch = matchVersions(appVersion, pkg.appVersion);
+        let versionsDoMatch = semver.satisfies(appVersion, pkg.appVersion);
```

Comment:
you need to do a similar update to the mariadb dao

Reply:
MariaDB doesn't require this semver validation. But the result is validated against the semver.